defaults/main.yml

```diff
@@ -1,5 +1,7 @@
 ---
 
+drush_path: /usr/local/bin/drush
```

Comment:
@seth-shaw-unlv Can you add `openseadragon_` to the front of this variable please

Reply:
Note that the geerlingguy.drupal and geerlingguy.drush roles set the drush_path for the environment, and we will want to use whatever they set it to be (which is why it works). 

So, if we change the local drush_path to openseadragon_drush_path in this role, then we should set openseadragon_drush_path to whatever the current drush_path value is, if it is set. Should that logic be done in tasks/main.yml or in defaults/main.yml ?

Reply:
So we are setting `drush_path` which is from `geerlingguy.drupal` and/or `geerlingguy.drush`? Or are they setting it?

My guess is to be _ansible compliant_ we should define a variable `openseadragon_drush_path` and then in the claw-playbook group_vars define `openseadragon_drush_path: {{ drush_path }}`

I think that should work.

Reply:
Currently, geerlingguy.drush is setting the variable and we are using what they set. So, taking your strategy (which is fine by me) we should state in the documentation that this variable needs to be set before it will work. I'll update that today.

Reply:
Slight modification. Keeping the openseadragon_drush_path default as it exists, but adding a new task to main to override it if drush_path is currently set.